sw/airborne/math/qr_solve/qr_solve.c

```diff
@@ -0,0 +1,1616 @@
+/*
```

Comment:
I don't know... I had to change a few things, and I suppose it is easier to compare with the original library from Burkardt if we don't change the code style